pandas/tests/indexes/datetimes/test_constructors.py

```diff
@@ -37,6 +37,14 @@
 
 
 class TestDatetimeIndex:
+    def test_from_dt64_unsupported_unit(self):
+        # GH#49292
+        val = np.datetime64(1, "D")
+        result = DatetimeIndex([val], tz="US/Pacific")
+
+        expected = DatetimeIndex([val.astype("M8[s]")], tz="US/Pacific")
```

Comment:
Is this actually what we expect? Any consideration to raising?

Reply:
This is correct.  In 2.0 we support resolutions of "s", "ms", "us", and "ns", and when we get something that isn't one of those we cast to the closest supported reso